TrueCraft/Commands/GiveCommand.cs

```diff
@@ -39,47 +38,59 @@ public override void Handle(IRemoteClient client, string alias, string[] argumen
             if(arguments.Length >= 4)
                     amount = arguments[3];
             
+            var receivingPlayer = GetPlayerByName(client, username);
+
+            if (receivingPlayer == null)
+            {
+                client.SendMessage("No client with the username \"" + username + "\" was found.");
```

Comment:
"client" isn't a very user-friendly term. How about "That player is not logged in"